### PR TITLE
{tools}[GCCcore/11.3.0] xxHash v0.8.1, python-xxhash v3.1.0 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by lexming**

### CODE REVIEW

File: easybuild/easyconfigs/p/python-xxhash/python-xxhash-3.1.0-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    ('Python', '3.10.4'),
+    ('xxHash', '0.8.1'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('xxhash', version, {
+        'checksums': ['ac21b1e21dc6fdfee9a57b53f4777539d53a84f2e1546a3f802f159f9966bdc1'],
+        'preinstallopts': 'XXHASH_LINK_SO=1',  # use xxHash library from EB
```
Comment (smoors) [inline easybuild/easyconfigs/p/python-xxhash/python-xxhash-3.1.0-GCCcore-11.3.0.eb:26]: @lexming should we add this to python-xxhash-3.2.0-GCCcore-12.2.0.eb as well?

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @lexming!

File: easybuild/easyconfigs/p/python-xxhash/python-xxhash-3.1.0-GCCcore-11.3.0.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/p/python-xxhash/python-xxhash-3.1.0-GCCcore-11.3.0.eb:26]: @lexming good point, done in https://github.com/easybuilders/easybuild-easyconfigs/pull/19432

**Pull request merged**